src/codecs/imagequant/options.tsx

```diff
@@ -26,50 +30,61 @@ export default class QuantizerOptions extends Component<Props, State> {
   @bind
   onChange(event: Event) {
     const form = (event.currentTarget as HTMLInputElement).closest('form') as HTMLFormElement;
+    const { options } = this.props;
 
-    const options: QuantizeOptions = {
-      zx: inputFieldValueAsNumber(form.zx),
-      maxNumColors: inputFieldValueAsNumber(form.maxNumColors),
+    const newOptions: QuantizeOptions = {
+      zx: inputFieldValueAsNumber(form.zx, options.zx),
```

Comment:
I've added a second "default value" parm to these. This is because the form fields may not be in the DOM anymore, because they're 'hidden'.